examples/polymer/vulcanize.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# vulcanize elements.html \
+#     --inline-script --inline-css --strip-comments | \
+#     crisper -h elements.build.html -j elements.build.js
+
+polybuild --maximum-crush elements/elements.html
```

Comment:
I think we should drop `vulcanize.sh` and just move this into an `npm run` script instead. It's a one liner that would fit well in package.json.